aarone_backend/contacts/tasks.py

```diff
@@ -0,0 +1,33 @@
+from celery import shared_task
+from django.template.loader import render_to_string
+from django.core.mail import EmailMessage
+from django.conf import settings
+
+@shared_task
+def send_email_task(instance_name, instance_email, instance_query):
+    # Send email notification with email template
+    subject = "Thanks for your message, we'll get back to you soon."
+    name_parts = instance_name.split()
+    first_name = "customer"
```

Comment:
Why is the `c` in `customer` not capitalized if we have code to capitalize their name if they entered it lower case?